pkg/asset/machines/openstack/machines.go

```diff
@@ -85,9 +85,12 @@ func provider(clusterID string, platform *openstack.Platform, mpool *openstack.M
 			APIVersion: openstackprovider.SchemeGroupVersion.String(),
 			Kind:       "OpenstackProviderSpec",
 		},
-		Flavor:         mpool.FlavorName,
-		CloudName:      CloudName,
-		CloudsSecret:   &corev1.SecretReference{Name: cloudsSecret, Namespace: cloudsSecretNamespace},
+		Flavor:       mpool.FlavorName,
+		CloudName:    CloudName,
+		CloudsSecret: &corev1.SecretReference{Name: cloudsSecret, Namespace: cloudsSecretNamespace},
+		// ConfigDrive type is *bool, so we can't use "true" directly and have to provide a pointer
+		// TODO(mfedosin):  change the type to bool
```

Comment:
nit: not sure if this is the most helpful todo since it refers to a change needed in a different repo, but agree that initializing a bool pointer w/ a value is tricky